modules/ocm-enabling-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,21 @@
+
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-enabling-autoscaling-nodes.adoc
+
+[id="ocm-enabling-autoscaling_{context}"]
+= Enabling autoscaling nodes in an existing cluster using OCM
+
+Enable autoscaling for worker nodes in the machine pool definition from the OCM console.
+
+.Procedure
+
+. From the link:https://console.redhat.com/[OCM console], navigate to the *Clusters* page and select the cluster for which you wish to enable autoscaling.
```

Comment:
"for which you wish to enable autoscaling" sounds off. Suggest rewording to something like "select the cluster that you want to enable autoscaling for" (it's fine to end in a preposition)